package.json

```diff
@@ -31,8 +31,8 @@
       "order-lines": "2.0 3.0",
       "orders": "11.0 12.0",
       "organizations.organizations": "1.0",
-      "pieces": "2.0",
-      "receiving": "1.0",
+      "pieces": "3.0",
```

Comment:
check interfaces in other places, inventory orders etc

Reply:
Sure.